mct_quantizers/keras/quantize_wrapper.py

```diff
@@ -300,7 +301,18 @@ def convert_to_inferable_quantizers(self):
                     if hasattr(quantizer, 'convert2inferable') and callable(quantizer.convert2inferable):
                         inferable_weight_quantizers.update({name: quantizer.convert2inferable()})
                 self.weights_quantizers = inferable_weight_quantizers
-                self._set_weights_vars(False)
+
+            # Create new layer with inferable quantizers
+            inferable_quantizers_wrapper = self.from_config(self.get_config())
+            inferable_quantizers_wrapper.layer.build(self.input_shape)
+            layer_weights_list = []
+            for weight_attr in self.weights_quantizers.keys():
+                layer_weights_list.append(getattr(self.layer, weight_attr)) # quantized weights
```

Comment:
Just a suggestion - if the attribute does not exist the call for getattr would not fail, but will return None which will then be inserted into the list. This can cause al sorts of weird behavior that would be very hard to debug.
Maybe assert that the returned attr is not None before appending it?